### ISSUE TITLE
Unable to add accessibility labels to react-native-material-ui components  for E2E testing

### ISSUE THREAD

**Issue by JPStrydom** [xotahal/react-native-material-ui#213]
The appium testing framework uses accessibility labels to identify and interact with elements. The current react-native-material-ui components however do not support these labels which makes automates E2E testing difficult and sometimes impossible.

**Comment by SitCode1993** [comment]
This would really help, I've also been struggling with assigning accessibility labels on the components as my e2e tests entirely depend on them. It would really be nice if I could be able to add accessibility labels

**Comment by kimibeyl** [comment]
+1

**Comment by ellipinsky** [comment]
+1

Has anybody figured out a workaround for this bug?  Or any ideas what a contributor needs to do to fix it?

**Comment by duenguyen2103** [comment]
+1 
Any workaround for the e2e testing labels?

**Comment by densanki** [comment]
It is extremely unpleasant this problem even after 2 years ticket run still occurs.
